src/main/java/kg/jarkyn/server/Outgoing/Response.java

```diff
@@ -0,0 +1,13 @@
+package kg.jarkyn.server.Outgoing;
+
+public class Response {
+    private final String body;
+
```

Comment:
I'm certain you'll eventually will need a `Response` object in your server, however right now this is just a very thin wrapper around String and doesn't add anything, it seems a bit unnecessary.